test-data.xml

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="pytest" errors="0" failures="0" skipped="0" tests="0" time="0.163" timestamp="2023-11-30T04:55:32.566267" hostname="renato-VivoBook-15-ASUS-Laptop-X540UAR" /></testsuites>
```

Comment:
This file, on the other hand, should be ignored. It records pytest runs in your machine.